src/app/introduction/activity-1/activity-1.component.ts

```diff
@@ -1,8 +1,11 @@
+
+import {interval as observableInterval} from 'rxjs';
+
+import {tap} from 'rxjs/operators';
 import { Component, OnInit, Output, EventEmitter, ViewContainerRef } from '@angular/core';
 import { Observable } from 'rxjs/Rx';
 import { DashboardService } from '../../services/dashboard.service';
 import { SharedDataService } from '../../services/shared.data.service';
-import { ToastsManager } from 'ng2-toastr/ng2-toastr';
```

Comment:
Yes! ng2-toastr did not support Angular 6. So, I had to move to ngx-toastr.

Reply:
In that case I don't see the import for ngx-toastr, I am guessing that it has been removed because its not being used here and we are using it in utility class?  Can you confirm?

Reply:
Yes! the same reason.